perfmetrics/scripts/ls_metrics/listing_benchmark.py

```diff
@@ -521,14 +539,15 @@ def _export_to_bigquery(test_type, config_id, start_time_build, ls_data):
 
   # If similar directory structure not found in the GCS bucket then delete all
   # the files in the bucket and make it from scratch.
-  if not directory_structure_present:
-    log.info(
-        """Similar directory structure not found in the GCS bucket.
-        Creating a new one.\n""")
-    log.info('Deleting previously present directories in the GCS bucket.\n')
-    subprocess.call(
-        'gsutil -m rm -r gs://{}/*'.format(directory_structure.name),
-        shell=True, stdout=subprocess.DEVNULL, stderr=subprocess.STDOUT)
+  print(directory_structure_present)
+  # if not directory_structure_present:
+  #   log.info(
+  #       """Similar directory structure not found in the GCS bucket.
```

Comment:
Have you create test data structure in bucket?

Reply:
yes.